charts/navidrome/0.0.1/test_values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for Navidrome.
+
+image:
+  repository: deluan/navidrome
+  pullPolicy: IfNotPresent
+  tag: 0.39.0
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 4533
+
+env:
+  ND_SCANINTERVAL: 15m
+  ND_LOGLEVEL: info
+  ND_SESSIONTIMEOUT: 24h
+  ND_ENABLETRANSCODINGCONFIG: "true"
+  ND_MUSICFOLDER: /music
+
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    mountPath: /data
+    accessMode: ReadWriteOnce
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    # storageClass: "-"
+    # accessMode: ReadWriteOnce
+    # size: 1Gi
+    ## Do not delete the pvc upon helm uninstall
+    # skipuninstall: false
+    # existingClaim: ""
+  music:
+    enabled: false
+    emptyDir: false
+    mountPath: /music
+    accessMode: ReadWriteOnce
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    # storageClass: "-"
+    # accessMode: ReadWriteOnce
+    # size: 1Gi
+    ## Do not delete the pvc upon helm uninstall
+    # skipuninstall: false
+    # existingClaim: ""
+
+
+appIngressEnabled: false
+    
+
+appVolumeMounts:
+  config:
+    enabled: true
+    emptyDir: true
+    mountPath: "/data"
+  music:
+    enabled: true
+    emptyDir: true
+    mountPath: "/music"
+    
```

Comment:
this is weird. how is there a new line (`71`) but still have no new line error?

Reply:
magic, I guess.